client/src/components/utils/Navbar/index.js

```diff
@@ -0,0 +1,153 @@
+    return (
+      <nav className={this.getNavbarClass(initialStatus, transparent, visible)}>
+        <div className={`${styles.container} container`}>
+          <div className={styles['image-container']}>
+            <Link to="/">
+              <img
+                className={styles.logo}
+                src={transparent ? logoImageLight : logoImageDark}
+                alt="logo"
+              />
+            </Link>
+          </div>
+          <div className={styles.menu}>
+            <ul className={styles.list}>
+              <li key="home" className={styles['list--item']}>
+                <NavLink
+                  exact
+                  className={`${
+                    styles['list--link']
+                  } ${this.getTransparentClass(transparent)}`}
+                  activeClassName={
+                    transparent
+                      ? ` ${styles['transparent--active']}`
+                      : styles.active
+                  }
+                  to="/"
+                >
+                  Home
+                </NavLink>
+              </li>
+              <li className={styles['list--item']}>
+                <NavLink
+                  exact
+                  className={`${
+                    styles['list--link']
+                  } ${this.getTransparentClass(transparent)}`}
+                  activeClassName={
+                    transparent
+                      ? ` ${styles['transparent--active']}`
+                      : styles.active
+                  }
+                  to="/about"
+                >
+                  About
+                </NavLink>
+              </li>
+              <li className={styles['list--item']}>
+                <NavLink
+                  exact
+                  className={`${
+                    styles['list--link']
+                  } ${this.getTransparentClass(transparent)}`}
+                  activeClassName={
+                    transparent
+                      ? ` ${styles['transparent--active']}`
+                      : styles.active
+                  }
+                  to="/sharing-buildings"
+                >
+                  Sharing Buildings
+                </NavLink>
+              </li>
+              <li className={styles['list--item']}>
+                <NavLink
+                  exact
+                  className={`${
+                    styles['list--link']
+                  } ${this.getTransparentClass(transparent)}`}
+                  activeClassName={
+                    transparent
+                      ? ` ${styles['transparent--active']}`
+                      : styles.active
+                  }
+                  to="/view-buildings"
+                >
+                  View Buildings
+                </NavLink>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+    );
+  }
+}
+
+Navbar.propTypes = {
+  transparent: PropTypes.bool.isRequired,
```

Comment:
```suggestion
  transparent: PropTypes.bool,
```
with a default value of `false`